example/string_generator.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python
+"""
+CherryPy String Generator Service.
+
+This is the example random string generator service, slightly
+modified for swagger to not use session support.
+"""
+import random
+import string
+import os
+import cherrypy
+
+STRING_FILE = 'string.txt'
```

Comment:
The only reason for this was to be able to support multiple processes, though its just an example so switching it to use in memory file would be fine.